WiserTaskScheduler/WiserTaskScheduler/Modules/Body/Models/BodyPartModel.cs

```diff
@@ -14,6 +14,23 @@ public class BodyPartModel
         /// </summary>
         [Required]
         public string Text { get; set; }
+        
+        /// <summary>
+        /// Gets or sets the item-id of the used template
+        /// </summary>
+        public ulong ItemId { get; set; }
+        
+        /// <summary>
+        /// Gets or sets the item-id of the used template
+        /// </summary>
+        public string LanguageCode { get; set; }
+
+        
```

Comment:
Please don't add unnecessary blank lines. A single blank line is enough here.

Reply:
Removed blank lines.